dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassPathResult.java

```diff
@@ -95,14 +95,15 @@ public String formatDependencyPaths(Iterable<ClassPathEntry> entries) {
   }
 
   /**
-   * Returns mapping from the Maven coordinates to class path entries that are in the dependency
-   * tree.
+   * Returns mapping from the Maven coordinates to {@link ClassPathEntry}. The keys are the
+   * coordinates of the direct dependencies of the root nodes in {@link #dependencyPaths}. The
+   * values are all {@link ClassPathEntry}s in the subtree of the key.
    */
-  public ImmutableSetMultimap<String, ClassPathEntry> coordinatesToClassPathEntry() {
+  public ImmutableSetMultimap<String, ClassPathEntry> coordinatesToClassPathEntriesInSubtree() {
```

Comment:
I'm not sure this method should be renamed like this. In general shorter names are easier to follow. Partly this is confusing because of the use of the ImmutableSetMultimap instead of a more specific class so you're trying to capture details in the method name that really belong in the return type.

Reply:
Reverted the renaming.

Right, I suffered from the vague type while I was debugging the purpose of this method. It tries to fetch all JAR files in a subtree of a Maven artifact but the method name was just rephrasing its return type. Updating Javadoc should suffice.